core-bundle/contao/languages/en/default.xlf

```diff
@@ -2270,6 +2270,21 @@
       <trans-unit id="MSC.skipToContent">
         <source>Skip to content</source>
       </trans-unit>
+      <trans-unit id="MSC.searchIndexer.0">
+        <source>Search indexer</source>
+      </trans-unit>
+      <trans-unit id="MSC.searchIndexer.1">
+        <source>Include or exclude this page from the search index (Contao Crawler).</source>
```

Comment:
```suggestion
        <source>Overrides the robots meta tag setting for the internal search index.</source>
```

wdyt? "Contao Crawler" does not need to be mentioned as it has nothing to do with the Contao Crawler directly.